### ISSUE TITLE
Suggestion: Add calendar group buttons to sidebar

### ISSUE THREAD

**Issue by BevansDesign** [OrestTa/MultipleCalendarsSelector#2]
It'd be great if you could add a way to switch between groups from the Calendar sidebar. I think that would be even more user-friendly than the current setup. (Plus I prefer not to add more icons to my pinned extensions bar.)

Here's a really rough mockup to demonstrate what I'm thinking of. You could use emojis instead of names to keep the buttons small.

<img width="204" alt="image" src="https://user-images.githubusercontent.com/296937/180478516-75358781-9f60-44e4-bc22-bde695ed818e.png">

I won't be offended if you don't add this, of course. 😄

Keep up the great work! I just found this extension about an hour ago, and it looks like it'll be pretty handy for me.

**Comment by OrestTa** [comment]
@BevansDesign absolutely, good idea. If you have some time to spare, feel free to implement something and create a PR, I'd be more than happy to merge :)
